packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -445,8 +453,9 @@ export class PerformanceEvent {
 		event: ITelemetryGenericEvent,
 		callback: (event: PerformanceEvent) => Promise<T>,
 		markers?: IPerformanceEventMarkers,
+		collectMemory?: boolean,
```

Comment:
#nit `collectMemory` as a parameter name sounds like we're collecting the memory itself? I don't have a good alternative. 'measureMemory` or `recordHeapSize` or `captureHeapSize`? No strong feelings one way or the other.

Reply:
recordHeapSize sounds much better!